api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -862,6 +864,20 @@ public void updateProfile_removeVerifiedInstitutionalAffiliationForbidden() {
     profileController.updateProfile(profile);
   }
 
+  @Test(expected = BadRequestException.class)
+  public void adminUpdateProfile_removeVerifiedInstitutionalAffiliationForbidden() {
```

Comment:
many of these tests are duplicates of existing updateProfile as admin versions

Reply:
⚠️ Do we want to allow the administrator to change every single field in the  Profile? Are there any fields that only a user should set (or be able to see)? I feel like that might be true today (and we should verify that), but I don't want to hang our hat on that assumption.